### PR TITLE
wrap new transform endpoint

### PR DESCRIPTION
**Pull request by yairwein**

### CODE REVIEW

**Comment by yairwein** [comment]
@guysp please review

File: alooma/alooma.py [hunk 1]
```diff
@@ -601,6 +601,21 @@ def set_transform(self, transform, module_name='main'):
         res = self.__send_request(requests.post, url, json=data)
         return res
 
+    def set_transform_with_modules(self, modules):
+        """
+        Set multiple transformation files.
```
Comment (guysp) [inline alooma/alooma.py]: for consistency, rename files -> modules?

File: alooma/alooma.py [hunk 2]
```diff
@@ -601,6 +601,21 @@ def set_transform(self, transform, module_name='main'):
...
+        modules has the following structure:
```
Comment (guysp) [inline alooma/alooma.py]: `:param modules:`?

File: alooma/alooma.py [hunk 3]
```diff
@@ -633,6 +648,40 @@ def test_transform(self, sample, temp_transform=None):
...
         return json.loads(res.content)
 
+    def test_transform_with_modules(self, sample, temp_modules):
+        """
+        :param sample:  a json string or a dict, representing a sample event
+        :param temp_modules: optional object containing transform code. if
+                        not provided, the currently deployed transform will be
+                        used. It is possible to override / add any number of
+                        modules. The modules that are not provided will be
+                        taken from the existing deployed code.
```
Comment (guysp) [inline alooma/alooma.py]: You are repeating the idea of "modules that are not provided"...
Maybe something like: 
```
optional object containing transform modules.
It is possible to override / add any number of
modules. The modules that are not provided will be
taken from the existing deployed code.
```

**Comment by oxmane** [comment]
Closing as irrelevant...

**Pull request closed without merging**